src/core/default/resources/simple/xlambda.py

```diff
@@ -345,6 +366,9 @@ def simple_function_annotation(
     name: str,
     events: List[Event] = [],
     environment={},
+    memory_size: int = 128,
+    timeout: int = 3,
```

Comment:
I know the aws lambda platform default is 3s, but lets make this 30s. I actually ran into some issues a while ago where my functions were timing out because of it being 3s.